### ISSUE TITLE
loss nan

### ISSUE THREAD

**Issue by Daisling** [NVlabs/RVT#55]
When I train rvt2, I get loss nan after 10 epochs, and I use config in model_99 ([pretrained RVT-2 model](https://drive.google.com/drive/folders/1yybWn6B7pGqbEi1t2hi_XPoELNgXaozY?usp=drive_link))
<img width="209" alt="785ffe0c68ab6cf73ebac0003b30e21" src="https://github.com/user-attachments/assets/4128ce40-6f61-47ed-b242-851a76d07e5d">
![image](https://github.com/user-attachments/assets/7226a6d4-f683-4d0b-9ee3-a7b2260693e8)
![image](https://github.com/user-attachments/assets/b9a00a9e-2cca-4e25-9074-06a550b81b40)

**Comment by EdenGabriel** [comment]
@Daisling Hi, I have the same question. Can u tell me how to solve it? Thank you very much.

**Comment by SmallStriker** [comment]
so what's the reason of loss Nan and how did you fix it finally?very thankful if you would like to share it.
